package.json

```diff
@@ -105,6 +105,7 @@
     "react-native-dotenv": "^0.2.0",
     "react-native-emoji": "^1.6.0",
     "react-native-extra-dimensions-android": "^1.2.5",
+    "react-native-fast-image": "^8.1.5",
```

Comment:
We're probably using proguard for release builds: https://github.com/DylanVann/react-native-fast-image#are-you-using-proguard

Reply:
yup 7723c36